pkg/types/clusterconfig/cluster_config_gcp.go

```diff
@@ -547,6 +576,8 @@ func (cc *GCPConfig) TelemetryEvent() map[string]interface{} {
 	if cc.ClusterName != "cortex" {
 		event["cluster_name._is_custom"] = true
 	}
+	event["preemptible"] = *cc.Preemptible
+	event["on_demand_backup"] = *cc.OnDemandBackup
```

Comment:
If these remain pointers, we should use the same convention as the other pointers (e.g. see `zone` below)